CHANGELOG.rst

```diff
@@ -1,5 +1,56 @@
 .. _changelog:
 
+0.10.0
+------
+
+This release includes the new user session controller in replacement of Jupyterhub, which will incur in breaking changes (notably, all user sessions will need to be closed) when upgrading to it.
+
+Improvements
+~~~~~~~~~~~~
+
+* **Documentation**: updated documentation on proper teaching etiquette and steps to use renkulab for teaching.
+* **User sessions**: use `Amalthea <https://github.com/SwissDataScienceCenter/amalthea>`__ to control sessions through a k8s operator.
```

Comment:
```suggestion
* **User sessions**: use `Amalthea <https://github.com/SwissDataScienceCenter/amalthea>`__ to control sessions through a k8s operator.
* **User sessions**: anonymous sessions do not require a separate namespace and renku-notebooks deployment, if enabled in the values file they now run in the same namespace as regular user sessions.
```

Reply:
thanks for the suggestion, I will add this to the ``Upgrading section`` for the admins.